include/termcolor/termcolor.hpp

```diff
@@ -202,6 +202,41 @@ namespace termcolor
         return stream;
     }
 
+    template <int code>
+    inline
+    std::ostream& fg_color(std::ostream& stream)
+    {
+        #if defined(TERMCOLOR_OS_MACOS) || defined(TERMCOLOR_OS_LINUX)
+        if (_internal::is_colorized(stream))
+        {
+            char command[14];
+            const char format[] = "\033[38;5;%dm"; 
```

Comment:
*(nitpicking)* I wonder, does it make sense to inline this constant into `std::sprintf`or better keep a separate symbol for it?

Reply:
Oh, I was suggesting something like:

```cpp
std::snprintf(command, 14, "\033[38;5;%dm", code);
```